aldryn_forms/contrib/email_notifications/cms_plugins.py

```diff
@@ -76,6 +93,15 @@ class EmailNotificationForm(FormPlugin):
         )
     ]
 
+    def get_inline_instances(self, request, obj=None):
+        inlines = super(EmailNotificationForm, self).get_inline_instances(request, obj)
+
+        if obj is None:
+            # remove the first inline which is ExistingEmailNotificationInline
+            # if we're first creating this object.
+            inlines = inlines[1:]
```

Comment:
If you specifically want to add that object, you should search for it and remove. Relying on its index in the list can be brittle and confusing.